package.json

```diff
@@ -1329,31 +1307,6 @@
         "view": "confluent-flink-database",
         "contents": "No Flink database (CCloud Kafka cluster) selected. Click below to get started.\n[Select Flink Database/Kafka Cluster](command:confluent.flinkdatabase.kafka-cluster.select)",
         "when": "config.confluent.flink.artifacts && confluent.ccloudConnectionAvailable && !confluent.flinkDatabaseSelected"
-      },
-      {
-        "view": "confluent-flink-database",
```

Comment:
TODO: we still need an empty state for the Flink Database view when search is applied